README.md

```diff
@@ -3,7 +3,8 @@
 
 # shellcheck-py
 
-A python wrapper to provide a pip-installable [shellcheck] binary.
+Run shellcheck in pre-commit even if the host machine doesn't have it installed!
```

Comment:
it also works even without pre-commit, (I find it way more convenient than fishing through the binaries page(s))

Reply:
ah yeah good point!